certified-connectors/Haiilo/readme.md

```diff
@@ -1,19 +1,19 @@
 
 
-## Smarp Connector  
-Smarp is a mobile-first employee communication platform to drive alignment, productivity and champion your brand. Connect to Smarp to manage your posts.  
+## Haiilo Advocacy Connector  
```

Comment:
Given that this is a significant alteration, could you kindly verify that this change will not impact any current flows or users? Additionally, please reach out to existing customers who have already established flows with this connector to notify them of the upcoming rebranding, as it represents a substantial modification.

Reply:
Hi, we've sent out communication-related to the branding update. Our customers are aware of the change and expecting it to be updated at anytime.